.github/workflows/additional.yml

```diff
@@ -10,6 +10,7 @@ defaults:
 jobs:
   mindeps:
     runs-on: "ubuntu-latest"
+    timeout-minutes: 90
```

Comment:
Is this something that could be set in `defaults` so that it doesn't need to be specified per-job?

Reply:
Ah, good suggestion -- I totally forgot that was an option. Let me push a commit for that...

Reply:
Hrm, it looks like either I'm using the wrong syntax or GitHub doesn't support this. We're getting [invalid workflow file errors](https://github.com/dask/dask/actions/runs/3942415512)

```
The workflow is not valid. .github/workflows/upstream.yml (Line: 10, Col: 3): Unexpected value 'timeout-minutes'
```

Any suggestions @charlesbluca? Otherwise, I might just revert back to `timeout-minutes` in each job separately

Reply:
Looks like this isn't supported yet https://github.com/orgs/community/discussions/25472. I've rolled back to specifying timeouts separately

Reply:
Sorry for the mislead here!